.env-dev-example

```diff
@@ -0,0 +1,4 @@
+DJANGO_SETTINGS_MODULE=cms.settings.dev 
```

Comment:
We should keep the extension as `.env` so I suggest renaming to `.development.env`. It is not just an example per-se, this can be used as is for development as is.